ng-dev/release/versioning/npm-command.ts

```diff
@@ -17,12 +17,20 @@ export abstract class NpmCommand {
    * Runs NPM publish within a specified package directory.
    * @throws With the process log output if the publish failed.
    */
-  static async publish(packagePath: string, distTag: NpmDistTag, registryUrl: string | undefined) {
-    const args = ['publish', '--access', 'public', '--tag', distTag];
-    // If a custom registry URL has been specified, add the `--registry` flag.
+  static async publish(
+    packagePath: string,
+    distTag: NpmDistTag | null,
```

Comment:
Does it make more sense to decouple publish and tagging? And we publish and then tag the newly published version, rather than having the login for whether or not to tag be in the publish logic?

Reply:
Good question. I believe the answer is that the decoupled tagging is already possible (we use it for lts tagging), but it doesn't hurt supporting here in the signature of publishing. Especially since it's the common case to set a tag when publishing (and the signature is also a great reminder to not forget about tagging)